### ISSUE TITLE
Focal plane building fails in the examples

### ISSUE THREAD

**Issue by keskitalo** [hpc4cmb/toast#216]
This issue probably came about from merging the focal plane building methods into TOAST proper:

Running toast_fake_focalplane.py...
OpenMP number of threads: 48
using 61 pixels (122 detectors)
Traceback (most recent call last):
  File "/scratch3/scratchdirs/keskital/software-toastdeps/bin/toast_fake_focalplane.py", line 93, in <module>
    Adets = tt.hex_layout(npix, width, angwidth, fwhm, "fake_", "A", Apol)
TypeError: hex_layout() takes from 5 to 6 positional arguments but 7 were given
srun: error: nid00124: task 0: Exited with exit code 1

**Comment by tskisner** [comment]
Yes, I fixed that in a branch that may not get merged, will open a separate PR for this.

**Comment by tskisner** [comment]
Although I fixed this, I note that the tools we have which are not really parallel (toast_fake_focalplane.py and toast_ground_schedule.py) die while dumping the timing profile.  However this only happens when doing "mpirun -n 1 <blah>" which is done in the example scripts.  Just running "blah" completes successfully.  I'm using mpich from the ubuntu packages.  In my changes in #217, I seemed to be able to work around this same problem by explicitly importing toast.MPI, and guarding the code inside of main() with a check for rank zero.

That is an unrelated problem from the focalplane interface though and is probably tightly coupled to the timing refactor, so I guess it does not make sense to worry about it until the dust settles on that work.  Just mentioning it before I forget.

PR coming in a second.
